### PR TITLE
Add visual regression testing

### PR DESCRIPTION
**Pull request by timstoddard**
Adds visual regression testing functionality to Playground via the all-powerful [toMatchImageSnapshot()](https://github.com/americanexpress/jest-image-snapshot) jest matcher.

New config options are as follows:
* __checkVisualRegressions__: boolean flag to run visual regression snapshot tests (default: `false`)
* __snapshotDirectory__: directory to store snapshots in (default: `'src/__images_snapshots__'`)
* __diffDirectory__: directory to store snapshots in (default: `'src/__diff_output__'`)
* __updateSnapshots__: boolean flag to update snapshots instead of testing them (default: `false`)
* __updateSnapshotsDirectory__: subdirectory of project to update snapshots only in there (default: `''`)
* __imageSnapshotConfig__: extra configuration for the matcher, details [here](https://github.com/americanexpress/jest-image-snapshot#optional-configuration) (default: `{}`), note this option can only be set via `angular-playground.json`

### CODE REVIEW

File: packages/angular-playground/cli/src/configure.ts [hunk 1]
```diff
@@ -52,6 +59,13 @@ export function configure(argv: any): Config {
         .option('--report-type <type>', 'Type of report to generate', REPORT_TYPE.LOG)
         .option('--report-path <path>', 'Path of report to generate', '')
 
+        // Snapshot tests
+        .option('--check-visual-regressions', 'Run visual regression tests', false)
+        .option('--snapshot-directory <dir>', 'Directory to store snapshots in', 'src/__images_snapshots__')
+        .option('--diff-directory <dir>', 'Directory to put diffs in', 'src/__diff_output__')
+        .option('--update-snapshots', 'Update stored snapshots', false)
+        .option('--update-snapshots-directory <div>', 'Subdirectory of project in which to update stored snapshots', '')
```
Comment (lurock) [inline packages/angular-playground/cli/src/configure.ts]: Change <div> to <dir>

Comment (timstoddard) [reply packages/angular-playground/cli/src/configure.ts]: test <div> test

Comment (timstoddard) [reply packages/angular-playground/cli/src/configure.ts]: <script>alert("hello");</script>

Comment (timstoddard) [reply packages/angular-playground/cli/src/configure.ts]: \<div>

**Pull request merged**